core/bug_api.php

```diff
@@ -314,7 +314,12 @@ public function __set( $p_name, $p_value ) {
 				break;
 			case 'due_date':
 				if( !is_numeric( $p_value ) ) {
-					$p_value = strtotime( $p_value );
+				    $p_value = DateTime::createFromFormat( config_get( 'normal_date_format' ), $p_value );
```

Comment:
A couple of questions to consider:
- Should be using db_null() or triggering an error?
- Should this method or another method return date_get_null() when date is not valid?